src/main/java/io/confluent/connect/elasticsearch/OffsetState.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2021 Confluent Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
@avocader any reason for Apache license?

Reply:
It was not intentional, updated to use `Confluent Community License`.